spec/features/admin/view_unconfirmed_users_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'support/notifications_service'
+
+describe 'viewing all unconfirmed users' do
+  include_examples 'notifications service'
+
+  let!(:admin_user) { create(:user, super_admin: true) }
+
+  let!(:organisation) { create(:organisation) }
+
+  let!(:unconfirmed_user1) { create(:user, confirmed_at: nil, email: "unconfirmed_user1@gov.uk") }
+  let!(:unconfirmed_user2) { create(:user, confirmed_at: nil, email: "unconfirmed_user2@gov.uk") }
+
+  let!(:confirmed_user1) { create(:user, confirmed_at: Time.now, email: "person1@gov.uk", organisation: organisation) }
+  let!(:confirmed_user2) { create(:user, confirmed_at: Time.now, email: "person2@gov.uk", organisation: organisation) }
```

Comment:
Do we need duplicates for the unconfirmed user

Reply:
Done